README.md

```diff
@@ -0,0 +1,33 @@
+# Luke's NC News Full Stack Project
+
+Welcome to my Full Stack project -
+https://lukesncnews.netlify.app/
+
+## The Project
+
+The site utilises my own Back End API to access development data. I then created an interactive news website where you can use filters, select topics, login, and post/delete comments.
```

Comment:
Try to flesh this out a bit. You want it to be a professional portfolio piece. Explain that this is the repository for the frontend of your nc-news project, which allows you to view articles, add comments.... etc.